{"name": "test_simple", "status": "failed", "statusDetails": {"message": "AssertionError: assert 4 == 5", "trace": "self = <test_login_page.TestLoginPage object at 0x7f0159c55a90>\n\n    def test_simple(self):\n        self.login_page.open()\n        self.login_page.enter_login(\"123\")\n>       assert 4 == 5\nE       assert 4 == 5\n\ntests/test_login_page.py:24: AssertionError"}, "start": 1736447525797, "stop": 1736447526649, "uuid": "5c0079c2-7a08-4576-89e2-c0e9ab7db937", "historyId": "b625b33df7b710c19b0b63ad85e53788", "testCaseId": "b625b33df7b710c19b0b63ad85e53788", "fullName": "tests.test_login_page.TestLoginPage#test_simple", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_page"}, {"name": "subSuite", "value": "TestLoginPage"}, {"name": "host", "value": "36df5e3aba09"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_page"}]}